### ISSUE TITLE
Lightkurve 2 API - normalize data not in flux column

### ISSUE THREAD

**Issue by orionlee** [KeplerGO/lightkurve#846]
#### Problem description
With Lightkurve2, if one wants to normalize data that is not `LightCurve.flux`, one has to jump through a few hoops, comparing to with Lightkurve 1.

#### Example

To normalize `SAP_FLUX`, in Lightkurve1, one can do it in 1 line

```python
    lc_normalized = lc.SAP_FLUX.normalize(unit='percent')
```

In Lightkurve2, one has to do it in a few steps (unless he/she uses the deprecated `.SAP_FLUX`)
```python
    lc_sap = lc.copy()
    lc_sap['flux] = lc_sap['sap_flux']
    lc_sap['sap_flux_err'] = lc_sap['sap_flux_err']
    lc_normalized = lc_sap.normalize(unit='percent')
```

#### Suggestion
Make `normalize`  takes an optional `column` attribute:

```python
    lc_normalized = lc.normalize(column='sap_flux', unit='percent')
```

The method gives a warning if the `{column}_err` does not exist.

**Comment by barentsen** [comment]
Agreed!  `lc.normalize()` could take an optional `column` argument.

Note: if a user wants to work with `SAP_FLUX`, my recommendation is to use
```python
lc = lk.read(filename, flux_column='sap_flux')`
```
or
```python
lk.search(...).download(flux_column='sap_flux')
```

This will automatically populate `flux` and `flux_err` with `sap_flux` and `sap_flux_err`.

**Comment by orionlee** [comment]
hmm, the having `flux_column` in `read` / `download` makes it less necessary to have `column` in `lc.normalize()` 

The `flux_column` in `read` / `download` is not yet (publicly) documented, however.

**Comment by barentsen** [comment]
Right!  I think adding a `column` parameter to `lc.normalize` could be a useful enhancement, e.g. we also added `lc.plot(column=...)`, but I don't think it is necessary to go ahead with v2.0.

Absolutely correct that the `flux_column` option is not well documented as this time.  Lots of work left to do re docs!
